kafe2/fit/_base/cost.py

```diff
@@ -1,4 +1,3 @@
-import abc
 import numpy as np
 import six
```

Comment:
```suggestion
import six
import warnings
```
For use later in this file instead of normal `print()`.